pages/blog/governance-motivation.md

```diff
@@ -0,0 +1,126 @@
+
+
+### Value work more than money
+
+We want to have a TSC consisting of all the active committers in the project, not people who are there because their company is a sponsor of the project. You are TSC member because you work on the project, you code, write docs, maintain CI. You work, and therefore, you are a part of the initiative.
+
+Why? Don't you like the money?
+
+We like money, don't get us wrong here :smiley: Money helps the project a lot, we can sponsor many good things with the money we obtain from you.
+
+Nevertheless, we believe that it isn't good to run a project alone with lots of bucks in the pocket. 
+
+What? We need a doctor here!
+
+:man_shrugging: We just prefer to work with a large group of people, from different cultures, from other parts of the world, with diverse experience and use cases in heads. That is just how you build great products by having many people around you to help and validate the ideas actively.
+
+
+### Respecting committers and the right to vote 
+
+The rule we want to follow is that the committer automatically gets a right to vote. 
+
+Why? 
+
+Voting is always about essential things, essential for the entire initiative. Who else should have a right to vote if not people that are directly affected by decisions of the ToC. The tricky situation here is that this approach can lead to a problem where one company hires most committers and, therefore, takes over project steering. This is something we actually have at the moment. The majority of folks working actively on the project are from Postman. Nothing terrible happened to the project for now, but this doesn't mean the whole community feels comfortable.
+
+We had a tough brainstorming here about finding the right balance between respecting all contributors equally and still making sure that there is no risk that a company has most voters. That is why we have a rule that only 1/4 (1/3 in case we need to look for more maintainers to bring balance back to the force) of voters can be affiliated with the same company. 
+
+### Work on all the tools under the AsyncAPI umbrella
+
+We want AsyncAPI Initiative to be a place where AsyncAPI open source tools are developed, together, so we do not duplicate each other in the community with different variations of the same tool. So far, seven AsyncAPI tools resulted from different companies or individuals' work and moved to the AsyncAPI GitHub organization. We want this to become a standard in the community. 
+
+The governance model embraces this direction. How? 
+
+If you own a project, you created it. It means you are its committer. 
+
+Do you see where I'm getting?
```

Comment:
Shall we write this in a way it represents us as a group?
```suggestion
Do you see where We're getting?
```

Reply:
definitely, as this blog post will be shared as by 2 authors, me and Franito